src/config/base.config.ts

```diff
@@ -1,5 +1,7 @@
 export default (): Record<string, unknown> => ({
   base: {
     port: process.env.PORT,
+    redis_host: process.env.REDIS_HOST,
+    redis_port: process.env.REDIS_PORT,
```

Comment:
We don't need `redis_host` and `redis_port` here, as we defined them into redis.config.ts